backend/addcorpus/validation/creation.py

```diff
@@ -186,3 +188,25 @@ def validate_sort_configuration(sort_config):
 
     if type(ascending) is not bool:
         raise ValidationError(f'Sort configuration has invalid "ascending" property: {ascending}')
+
+
+def validate_source_data_configuration(source_data: Dict) -> None:
```

Comment:
Please provide a docstring. What are expected inputs? In which cases should the ValidationError be raised?